packages/core-graphql/__tests__/graphql.test.js

```diff
@@ -21,63 +31,32 @@ describe('GraphQL', () => {
     expect(graphql).toBeObject()
   })
 
-  describe('test queries', () => {
+  describe('test data', () => {
     it('should be an object', () => {
-      expect(testQueries).toBeObject()
+      expect(testData).toBeObject()
     })
 
-    describe('test query for Block', () => {
-      it('should have queries', () => {
-        expect(testQueries.block.length).toBeTruthy()
+    describe('loading blocks', () => {
+      references.data.blocks = testData.blocks
+      expect(references.data.blocks).toBe(testData.blocks)
```

Comment:
What does this achieve ? 
Also, any `expect` should be inside some `it` block, I don't really see what it is doing here.